docs/testing/raid-test-plan.md

```
@@ -0,0 +1,1281 @@
+
+**Setup**: Create raid1 with 2 devices + PVC.
+
+### Test 6A.1: Healthy Metrics
+
+```bash
+VG_POD=$(oc get pods -n $NS -l app.kubernetes.io/name=vg-manager --field-selector spec.nodeName=$NODE -o jsonpath='{.items[0].metadata.name}')
+oc exec -n $NS $VG_POD -- curl -s http://localhost:8080/metrics | grep lvms_raid
+```
+
+**Expected:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)